cmd/controller-manager/main.go

```diff
@@ -145,6 +149,9 @@ func main() {
 	go kubeInformerFactory.Start(controllerCtx.Done())
 
 	onStarted := func(ctx context.Context) {
+		if err := generateClientCert(ns, "tidb-operator", certController); err != nil {
```

Comment:
I recommend creating the operator client certificate in the tidb-operator installation.